collectors/python.d.plugin/gearman/gearman.chart.py

```diff
@@ -0,0 +1,157 @@
+# Description: dovecot netdata python.d module
+# Author: Kyle Agronick (agronick)
+# SPDX-License-Identifier: GPL-3.0+
+
+# Gearman Netdata Plugin
+
+from bases.FrameworkServices.SocketService import SocketService
+from copy import deepcopy
+
+update_every = 3
+priority = 60500
+
+
+CHARTS = {
+    'total_workers': {
+        'options': ['total', 'Total Workers', 'Workers', 'Total Workers', 'gearman.total', 'line'],
```

Comment:
> gearman.total

keep in mind that context used as a metric name when exporting to prometheus for example, so i think we need to change the name

Reply:
let me give you an example

https://github.com/netdata/netdata/blob/2fbc25e1b08959e49e9ece4e816f2ba41a863f84/collectors/python.d.plugin/sensors/sensors.chart.py#L23-L28

context is `sensors.temperature`, as you can see it makes sense, `gearman.total` is not something clear, total what?

Back to `sensors.temperature`. Keep in mind that netdata is an agent and people use it to collect metrics and export them to some db. Netdata export metrics.

```
netdata_sensors_temperature_Celsius_average{chart="sensors.coretemp_isa_0000_temperature",family="temperature",dimension="Core 0"} 36.2000000 1565853021000
netdata_sensors_temperature_Celsius_average{chart="sensors.coretemp_isa_0000_temperature",family="temperature",dimension="Core 1"} 36.8000000 1565853021000
netdata_sensors_temperature_Celsius_average{chart="sensors.coretemp_isa_0000_temperature",family="temperature",dimension="Core 2"} 35.8000000 1565853021000
netdata_sensors_temperature_Celsius_average{chart="sensors.coretemp_isa_0000_temperature",family="temperature",dimension="Core 3"} 36.2000000 1565853021000
netdata_sensors_temperature_Celsius_average{chart="sensors.coretemp_isa_0000_temperature",family="temperature",dimension="Core 4"} 36.0000000 1565853021000
netdata_sensors_temperature_Celsius_average{chart="sensors.coretemp_isa_0000_temperature",family="temperature",dimension="Core 5"} 35.1000000 1565853021000
```

The core element in metrics name is context - `sensors_temperature`. So this is crucial to have good context, because the context is metric name actually.

Reply:
ok, in your case metric name will be

`netdata_gearman_total_Workers`

I [see](https://prometheus.io/docs/practices/naming/#metric-names) the valid order is
 - agent prefix
 - app prefix
 - metric name
 - metric unit

In your case `total` is metric name, that is why i am complaining :sweat_smile: